server/clip_server/executors/helper.py

```diff
@@ -4,7 +4,7 @@
 from docarray import Document, DocumentArray
 from docarray.math.distance.numpy import cosine
 
-from clip_server.model import clip
+from ..model import clip
```

Comment:
please keep original

Reply:
Is there any strong reasons to use absolute import path? The relative import is used here to address the import issue for hub executor (as recommende at https://docs.jina.ai/fundamentals/executor/executor-files/#multiple-python-files-yaml).